test/foundry/AuthorizedReceiptTransfer.t.sol

```diff
@@ -0,0 +1,113 @@
+// SPDX-License-Identifier: CAL
+pragma solidity =0.8.17;
+
+import {CreateOffchainAssetReceiptVaultFactory} from "../../contracts/test/CreateOffchainAssetReceiptVaultFactory.sol";
+import {Test, Vm} from "forge-std/Test.sol";
+import {
+    OffchainAssetReceiptVault,
+    CertificationExpired
+} from "../../contracts/vault/offchainAsset/OffchainAssetReceiptVault.sol";
+import {OffchainAssetVaultCreator} from "./OffchainAssetVaultCreator.sol";
+
+contract AuthorizedReceiptTransfer is Test, CreateOffchainAssetReceiptVaultFactory {
+    ///Test AuthorizeReceiptTransfer reverts if system not certified
+    function testAuthorizeReceiptTransferRevert(
+        uint256 fuzzedKeyAlice,
+        uint256 fuzzedKeyBob,
+        uint256 warpTimestamp,
+        string memory assetName,
+        string memory assetSymbol
+    ) external {
+        // Ensure the fuzzed key is within the valid range for secp256k1
+        fuzzedKeyAlice = bound(fuzzedKeyAlice, 1, SECP256K1_ORDER - 1);
+        address alice = vm.addr(fuzzedKeyAlice);
+
+        // Ensure the fuzzed key is within the valid range for secp256k1
+        fuzzedKeyBob = bound(fuzzedKeyBob, 1, SECP256K1_ORDER - 1);
+        address bob = vm.addr(fuzzedKeyBob);
+
+        warpTimestamp = bound(warpTimestamp, 1, 1e6);
+
+        OffchainAssetReceiptVault vault = OffchainAssetVaultCreator.createVault(factory, alice, assetName, assetSymbol);
+
+        // Warp the block timestamp to a non-zero value
+        vm.warp(warpTimestamp);
+
+        // Use the same timestamp in the expectRevert check
+        vm.expectRevert(abi.encodeWithSelector(CertificationExpired.selector, alice, bob, 0, warpTimestamp));
```

Comment:
"not certified" doesn't just mean that the timestamp is `0` (i.e. never certified), it also means that the certification is not zero but in the past, so expired

why aren't we warping and fuzzing the expiry time?